deepspeed/__init__.py

```diff
@@ -56,7 +56,8 @@ def initialize(args,
         mpu: Optional: A model parallelism unit object that implements
             get_{model,data}_parallel_{rank,group,world_size}()
 
-        dist_init_required: Optional: Initializes torch.distributed
+        dist_init_required: Optional: None will auto-initialize torch.distributed if needed,
```

Comment:
Are we officially deprecating this argument? If so, it would be good to add a "(deprecated)" to the docstring.

Reply:
Talked offline, waiting to deprecate for now.